api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst

```diff
@@ -0,0 +1,243 @@
+.. _get-list-users-with-role-v2.0:
+
+List users with role
+~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v2.0/OS-KSADM/roles/{roleId}/RAX-AUTH/users
+
+This operation returns a list of users with detailed account information about
+each user, including email, name, user ID, phone pin state, account
+configuration, and status information assigned to the specified role ID.
+
+.. note::
+
+  - Users with the ``identity:service-admin`` role can list users with the
```

Comment:
usually, we do not expose ``identity:service-admin``  info in public-facing docs. @kidster can confirm if we need it here or not.

Reply:
Yes, that is correct. We need to remove the sections of `identity:service-admin` and `identity:admin`. Public docs should only show information relevant to a customer.